actions/st2_chg_ver_for_st2_dockerfiles.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+set -e
+
+# When there's a st2 release:
+#  - if missing, create a branch named ${BRANCH} (i.e., "vX.Y") in ${PROJECT}.
+#  - update the Makefile on this branch to reference $ST2_VERSION (i.e., "X.Y.Z").
+# Another script will update the master branch with the next dev release.
+
+FORK=$1
+PROJECT=$2
+ST2_VERSION=$3
+BRANCH=$4
+
+GIT_REPO="git@github.com:${FORK}/${PROJECT}.git"
+CWD=`pwd`
+
+echo "Checking existence of branch ${BRANCH}"
+BRANCH_EXISTS=`git ls-remote --heads ${GIT_REPO} | grep refs/heads/${BRANCH} || true`
+
+if [[ -z ${LOCAL_REPO} ]]; then
+    CURRENT_TIMESTAMP=`date +'%s'`
+    RANDOM_NUMBER=`awk -v min=100 -v max=999 'BEGIN{srand(); print int(min+rand()*(max-min+1))}'`
+    LOCAL_REPO=${PROJECT}_${CURRENT_TIMESTAMP}_${RANDOM_NUMBER}
+fi
+
+if [ -d "${LOCAL_REPO}" ]; then
+    echo "Deleting ${LOCAL_REPO}"
+    rm -rf ${LOCAL_REPO}
+fi
+
+echo "Cloning repository to ${LOCAL_REPO}"
+git clone ${GIT_REPO} ${LOCAL_REPO}
+
+cd ${LOCAL_REPO}
+echo "Current directory: `pwd`..."
+
+if [[ -z "${BRANCH_EXISTS}" ]]; then
+    echo "Creating branch ${BRANCH}"
+    git branch ${BRANCH}
+fi
+
+echo "Checking out ${BRANCH}"
+git checkout ${BRANCH}
+
+echo "Updating Makefile"
+# Replace line in Makefile beginning "ST2_VERSION ?=" with "ST2_VERSION ?= ${ST2_VERSION}"
+sed -i "/^ST2_VERSION.*/cST2_VERSION ?= ${ST2_VERSION}" Makefile
```

Comment:
Need better error reporting if change failed for some reason, one of the examles:
https://github.com/StackStorm/st2cd/blob/11de09c4f16cc699e70535ed4cd29cf5dbbf6ba4/actions/st2_chg_ver_for_st2.sh#L116-L120